faces/3.0/_index.md

```diff
@@ -9,12 +9,15 @@ support for internationalization and accessibility.
 
 * [Jakarta Server Faces 3.0 Release Record](https://projects.eclipse.org/projects/ee4j.faces/releases/3.0.0)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta Server Faces 3.0 Specification Document](./faces-3.0-SNAPSHOT.pdf) (PDF)
-* [Jakarta Server Faces 3.0 Specification Document](./faces-3.0-SNAPSHOT.html) (HTML)
+* [Jakarta Server Faces 3.0 Specification Document](./jakarta-faces-3.0.pdf) (PDF)
+* [Jakarta Server Faces 3.0 Specification Document](./jakarta-faces-3.0.html) (HTML)
 * [Jakarta Server Faces 3.0 Javadoc](./apidocs)
-* [Jakarta Server Faces 3.0 TCK]()([sig](),[sha](),[pub]())
+* [Jakarta Server Faces 3.0 Jsdoc](./jsdoc)
+* [Jakarta Server Faces 3.0 Renderkitdoc](./renderkitdoc)
+* [Jakarta Server Faces 3.0 VDLDoc](./vdldoc)
+* [Jakarta Server Faces 3.0 TCK](https://download.eclipse.org/jakartaee/faces/jakarta-faces-tck-3.0.0.zip)([sig](https://download.eclipse.org/jakartaee/faces/jakarta-faces-tck-3.0.0.zip.sig),[sha](https://download.eclipse.org/jakartaee/faces/jakarta-faces-tck-3.0.0.zip.sha256),[pub](https://raw.githubusercontent.com/jakartaee/specification-committee/master/jakartaee-spec-committee.pub))
```

Comment:
```suggestion
* [Jakarta Server Faces 3.0 TCK](https://download.eclipse.org/jakartaee/faces/3.0/jakarta-faces-tck-3.0.0.zip)([sig](https://download.eclipse.org/jakartaee/faces/3.0/jakarta-faces-tck-3.0.0.zip.sig),[sha](https://download.eclipse.org/jakartaee/faces/3.0/jakarta-faces-tck-3.0.0.zip.sha256),[pub](https://raw.githubusercontent.com/jakartaee/specification-committee/master/jakartaee-spec-committee.pub))
```
Missing the version in your URLs, as per checklist `Link to final TCK download zip file of the form https://download.eclipse.org/jakartaee/{spec}/x.y/*{spec}-tck-x.y.z.zip (The folder path is required, the file name pattern is preferred.) `